netdeploy/remote/deployedNetwork.go

```diff
@@ -62,13 +73,48 @@ type DeployedNetworkConfig struct {
 
 // DeployedNetwork represents the complete configuration specification for a deployed network
 type DeployedNetwork struct {
-	useExistingGenesis bool
+	useExistingGenesis       bool
+	createBoostrappedNetwork bool
+	GenesisData              gen.GenesisData
+	Topology                 topology
+	Hosts                    []HostConfig
+	BootstrappedNet          BootstrappedNetwork
+}
+
+type netState struct {
+	nAccounts     uint64
+	nAssets       uint64
+	nApplications uint64
+	roundTxnCnt   uint64
+
+	assetPerAcct int
+	appsPerAcct  int
+
+	genesisID   string
+	genesisHash crypto.Digest
+	poolAddr    basics.Address
+	sinkAddr    basics.Address
+
+	accountsCreated bool
+	txnState        protocol.TxType
 
-	GenesisData gen.GenesisData
-	Topology    topology
-	Hosts       []HostConfig
+	round          basics.Round
+	accounts       []basics.Address
+	trxCnt         uint64
+	fundPerAccount basics.MicroAlgos
 }
 
+const program = `#pragma version 2
+txn ApplicationID
+bz ok
+int 0
+byte "key"
+byte "value"
+app_local_put
+ok:
+int 1
+`
```

Comment:
This probably need also to go into the `BootstrappedNetwork` structure. But we can defer this to a subsequent task.